styles/index.css

```diff
@@ -0,0 +1,148 @@
+#top {
+    text-align : center;
+}
+body {
+    background-image: linear-gradient(to right, blue, rgb(0, 128, 255), white) ;
+    font-family:Georgia, 'Times New Roman', Times, serif;
+}
+
+input{
+    padding:5px;
+    font-size:16px;
+    font-family:Georgia, 'Times New Roman', Times, serif;    
+}
+
+select {
+    padding:5px;
+    font-size:16px;
+    font-family:Georgia, 'Times New Roman', Times, serif;    
+}
+
+
+button {
+    padding:5px;
+    font-size:16px;
+    font-family:Georgia, 'Times New Roman', Times, serif;    
+}
+
+.boxes {
+    display: grid;
+    grid-template-columns: repeat(10,10%);
+    grid-template-rows: repeat(5,20%);
+}
+
+.flex_boxes_container {
+    background-color: antiquewhite;
+    grid-column : 2 /span 3;
+    grid-row: 1/ -1;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    
+}
+
+.flex_boxes {
+    border: 10px  rgb(0, 128, 255) solid;
+    padding: 10px ;
+    text-align: center;
+}
+
+#tempurature_box {
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+}
+
+#weather_garden_box {
+    grid-column : 7 /span 3;
+    grid-row: 2/ span 3;
+    border: 1px solid #000;
+    text-align: center;
+    display: grid;
+    grid-template-columns: repeat(4,25%);
+    grid-template-rows: repeat(4,25%);
+    background-color: antiquewhite;
+}
+
+
+#weather_visuals {
+    /* background-color: rgb(131, 255, 127); */
+    grid-row: 2/ -1;
+    grid-column: 1/ -1;
+    display: grid;
+    grid-template-columns: repeat(4,25%);
+    grid-template-rows: repeat(4,25%);
+}
+
+#title{
+    grid-column: 1/-1;
+    grid-row: 1/2;
+    text-align: center;
+}
+
+#skyEmojis {
+    grid-row: 1/2;
+    grid-column: 1/-1;
+    font-size: xx-large;
+}
+
+#landscape {
+    grid-row: 4/-1;
+    grid-column: 1/-1;
+    font-size: x-large;
+}
+
+.cold {
+    color : teal;
+}
+
+.hot {
+    color : red;
+}
+
+.warm {
+    color : orange;
+}
+
+.cozy {
+    color : rgb(215, 206, 32);
+}
+
+.cool {
+    color : green;
+}
+
+.sun {
+    background-image: url(https://www.vmcdn.ca/f/files/sudbury/uploadedImages/news/localNews/2019/07/070719_kf_sun_sized.jpg;w=960);
+    background-position: center;
+    background-size: 100%;
+}
+
+.clouds {
+    background-image: url(https://www.louisdallaraphotography.com/wp-content/uploads/2019/11/clouds-photo-6119.jpg);
+    background-position: center;
+    background-size: 100%;
+}
+
+.rain {
+    background-image: url(https://i.pinimg.com/originals/44/db/6e/44db6e6fb688ab769d4fc203632ad190.jpg);
+    background-position: center;
+    background-size: 100%;
```

Comment:
The `background-position` and `background-size` are repeated across a lot of your classes. Can you consider how you could make this more DRY?